independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/app/CuratedApplication.java

```diff
@@ -147,7 +147,9 @@ private synchronized void processCpElement(AppArtifact artifact, Consumer<ClassP
         }
         artifact.getPaths().forEach(path -> {
             final ClassPathElement element = ClassPathElement.fromPath(path);
-            augmentationElements.put(artifact, element);
+            if (artifact.getPaths().isSinglePath()) {
```

Comment:
what's the idea behind this change?

Reply:
If there are two paths, then the second time only one gets added, as they both have the same map key.

Reply:
oh, i missed that. that's not good.